presto-jdbc/src/main/java/io/prestosql/jdbc/AbstractPrestoResultSet.java

```diff
@@ -1750,6 +1750,9 @@ private static Number toNumber(Object value)
         if (value instanceof Boolean) {
             return ((Boolean) value) ? 1 : 0;
         }
+        if (value instanceof String) {
+            return new BigDecimal((String) value);
```

Comment:
Add a test case where `rs.get{Long,Double,BigDecimal,..}` are called on a value that is not a number:

- `''`
- `'123a'`

Here we probably will need to catch the exception coming from BigDecimal constructor so that we reach `throw new SQLException("Value is not a number:` line

Reply:
Sure.